tests/test_memcache_ketama_client.py

```diff
@@ -0,0 +1,134 @@
+from unittest import TestCase
+import random
+import string
+
+import memcache
+
+from memcache_ketama_client import MemcacheKetamaClient
+from collections import defaultdict
+
+
+class MemcacheClientTestCase(TestCase):
+    def setUp(self):
```

Comment:
use setUpClass so it its done once for all test.. and in setUp you can just clean out the db